bin/time_rhs.py

```diff
@@ -37,14 +36,15 @@
 p_dct = dict(zip(constants, p_array))
 
 p = {}
-p[None] = choice([p_array, p_dct])
+# for p[None] 'array' argument type has been assumed
+p[None] = p_array
```

Comment:
Maybe just don't do the runs for None at all since it is the same thing as the line below it.